src/main/java/pb360/data/entity/RestApi.java

```diff
@@ -0,0 +1,85 @@
+package pb360.data.entity;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import pb360.model.MessageObject;
+
+@Document(collection= "RestApi")
+
+public final class RestApi {
+	private @JsonProperty("restId") String restId;
+
+	private @JsonProperty("restName") String restName;
+
+	private @JsonProperty("restUrl") String restUrl;
+
+	private @JsonProperty("restStatus") MessageObject restStatus;
+
```

Comment:
remove @JsonProperty() in the define variable scope, only add it on the constructor